templates/course/course.html

```diff
@@ -1,26 +1,26 @@
 <!-- course.html -->
 
 
-<form method="GET">
+<form  method="GET">

```

Comment:
```suggestion
<form method="GET">
```
There should be only one space.